package.json

```diff
@@ -1,78 +1,52 @@
+  "version": "0.0.0",
   "scripts": {
-    "start": "webpack-dev-server --port 9002 --content-base app --config app/webpack.config --open",
-    "lint": "npm run tslint 'src/**/*.ts' 'app/**/*.ts'",
-    "clean": "rimraf dist",
-    "build": "npm-run-all --serial lint clean build:ngc build:umd build:app",
-    "build:ngc": "ngc -p tsconfig.ngc.json",
-    "build:umd": "webpack --mode development",
-    "build:app": "webpack --mode production --config app/webpack.config",
-    "build-and-publish": "npm run build && npm version patch && npm publish --access=public",
-    "tslint": "tslint"
+    "ng": "ng",
+    "start": "ng serve",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e",
+    "build-lib:watch": "ng build auto-complete --watch",
+    "build-lib:prod": "ng build auto-complete --prod"
   },
+  "private": true,
   "dependencies": {
-    "@ngui/utils": "^0.8.1"
+    "@angular/animations": "~9.0.0",
+    "@angular/cdk": "^9.1.0",
+    "@angular/common": "~9.0.0",
+    "@angular/compiler": "~9.0.0",
+    "@angular/core": "~9.0.0",
+    "@angular/forms": "~9.0.0",
+    "@angular/material": "^9.1.0",
+    "@angular/platform-browser": "~9.0.0",
+    "@angular/platform-browser-dynamic": "~9.0.0",
+    "@angular/router": "~9.0.0",
+    "rxjs": "~6.5.4",
+    "tslib": "^1.10.0",
+    "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular/animations": "^6.0.7",
-    "@angular/cdk": "^6.3.2",
-    "@angular/cli": "^6.0.8",
-    "@angular/common": "^6.0.7",
-    "@angular/compiler": "^6.0.7",
-    "@angular/compiler-cli": "^6.0.7",
-    "@angular/core": "^6.0.7",
-    "@angular/forms": "^6.0.7",
-    "@angular/material": "^6.3.2",
-    "@angular/platform-browser": "^6.0.7",
-    "@angular/platform-browser-dynamic": "^6.0.7",
-    "@angular/router": "^6.0.7",
-    "@ngtools/webpack": "~6.0.8",
-    "@types/hammerjs": "^2.0.35",
-    "@types/node": "^10.5.2",
-    "@types/uglify-js": "^3.0.2",
-    "@types/webpack": "^4.4.5",
-    "angular2-template-loader": "^0.6.2",
-    "awesome-typescript-loader": "~5.2.0",
-    "codelyzer": "^4.4.2",
-    "core-js": "^2.5.7",
-    "forever": "^0.15.3",
-    "hammerjs": "^2.0.8",
-    "http-server": "^0.11.1",
-    "npm-run-all": "^4.1.3",
-    "raw-loader": "^0.5.1",
-    "reflect-metadata": "^0.1.12",
-    "rimraf": "^2.6.2",
-    "rxjs": "^6.2.1",
-    "strip-loader": "^0.1.2",
-    "systemjs": "~0.21.4",
-    "ts-loader": "^4.4.2",
-    "tslint": "^5.10.0",
-    "typescript": "~2.7.2",
-    "webpack": "^4.15.1",
-    "webpack-cli": "~3.0.8",
-    "webpack-dev-server": "~3.1.4",
-    "zone.js": "~0.8.26"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/ng2-ui/auto-complete.git"
-  },
-  "author": "",
-  "bugs": {
-    "url": "https://github.com/ng2-ui/auto-complete/issues"
-  },
-  "homepage": "https://github.com/ng2-ui/auto-complete#readme",
-  "keywords": [
-    "angular",
-    "auto-complete",
-    "input",
-    "select"
-  ]
```

Comment:
Please keep these.

Reply:
It should be fixed. But refer - `auto-complate/projects/auto-complete/package.json` file.